### PR TITLE
Fixing warning level read/write logic

### PR DESCRIPTION
**Pull request by mishra14**
Fixing couple of issues with warning levels - 

1. Changing the write api to write the warning level as an int to make it compatible with .Net standard warning levels.

2. Changing the default behavior for read/writes.

### CODE REVIEW

File: test/NuGet.Core.Tests/NuGet.ProjectModel.Test/LockFileFormatTests.cs [hunk 1]
```diff
@@ -606,13 +606,13 @@ public void LockFileFormat_WritesMultipleErrorMessages()
...
       ""level"": ""Warning"",
-      ""warningLevel"": ""Severe"",
+      ""warningLevel"": ""1"",
       ""message"": ""test log message""
     },
     {
       ""code"": ""NU1000"",
       ""level"": ""Warning"",
-      ""warningLevel"": ""Severe"",
+      ""warningLevel"": ""2"",
```
Comment (emgarten) [inline test/NuGet.Core.Tests/NuGet.ProjectModel.Test/LockFileFormatTests.cs]: this seems like it should be an int instead of a string

**Review by emgarten — CHANGES_REQUESTED** [review]
string -> int

**Comment by mishra14** [comment]
🔔

**Review by emgarten — APPROVED** [review]
🥇

**Pull request merged**